pkg/k8shandler/kibana/reconciler.go

```diff
@@ -74,6 +110,10 @@ func ReconcileKibana(requestCluster *kibana.Kibana, requestClient client.Client,
 		return err
 	}
 
+	if err := migrationRequest.RunKibanaMigrations(); err != nil {
```

Comment:
should we also ensure there is an ES cluster for us to connect to?

Reply:
Since we check the index health before running any migrations i would say we already cover this case.